### ISSUE TITLE
 Can RarWriter be added ？

### ISSUE THREAD

**Issue by a764578566** [adamhathcock/sharpcompress#401]
![image](https://user-images.githubusercontent.com/14343051/42488419-00c7550a-8438-11e8-8c62-97ed449a414f.png)

**Comment by adamhathcock** [comment]
No.  RAR is closed source while UnRAR is open source.  The only way to make rar files is with WinRAR or other tooling made by the same guy.
